[2022-08-18 18:55:03,694] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: elastic_desk.popula_elastic_finalizados scheduled__2022-08-18T18:50:00+00:00 [queued]>
[2022-08-18 18:55:03,716] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: elastic_desk.popula_elastic_finalizados scheduled__2022-08-18T18:50:00+00:00 [queued]>
[2022-08-18 18:55:03,716] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-18 18:55:03,716] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-18 18:55:03,716] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-18 18:55:03,747] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): popula_elastic_finalizados> on 2022-08-18 18:50:00+00:00
[2022-08-18 18:55:03,753] {standard_task_runner.py:52} INFO - Started process 12532 to run task
[2022-08-18 18:55:03,763] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'elastic_desk', 'popula_elastic_finalizados', 'scheduled__2022-08-18T18:50:00+00:00', '--job-id', '82', '--raw', '--subdir', 'DAGS_FOLDER/etl_elastic_desk.py', '--cfg-path', '/tmp/tmpxji5iefn', '--error-file', '/tmp/tmpdfqkmq8b']
[2022-08-18 18:55:03,764] {standard_task_runner.py:80} INFO - Job 82: Subtask popula_elastic_finalizados
[2022-08-18 18:55:03,879] {task_command.py:371} INFO - Running <TaskInstance: elastic_desk.popula_elastic_finalizados scheduled__2022-08-18T18:50:00+00:00 [running]> on host 1e1070d9e5ae
[2022-08-18 18:55:03,993] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clecio.antao@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=elastic_desk
AIRFLOW_CTX_TASK_ID=popula_elastic_finalizados
AIRFLOW_CTX_EXECUTION_DATE=2022-08-18T18:50:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-18T18:50:00+00:00
[2022-08-18 18:55:04,000] {base.py:271} INFO - GET http://192.168.2.100:9200/ [status:200 request:0.005s]
[2022-08-18 18:55:07,819] {base.py:271} INFO - HEAD http://192.168.2.100:9200/_template/desk_finalizados [status:200 request:0.006s]
[2022-08-18 18:55:08,329] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/es_pandas/es_pandas.py:58 RuntimeWarning: invalid value encountered in double_scalars
[2022-08-18 18:55:08,685] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.176s]
[2022-08-18 18:55:09,060] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.290s]
[2022-08-18 18:55:09,371] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.079s]
[2022-08-18 18:55:09,657] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.133s]
[2022-08-18 18:55:10,233] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.186s]
[2022-08-18 18:55:10,736] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.132s]
[2022-08-18 18:55:10,944] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.052s]
[2022-08-18 18:55:11,165] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.086s]
[2022-08-18 18:55:11,334] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.078s]
[2022-08-18 18:55:11,397] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.044s]
[2022-08-18 18:55:11,479] {logging_mixin.py:115} INFO - 0       18-08-2022
1       18-08-2022
2       18-08-2022
3       18-08-2022
4       18-08-2022
           ...    
4768    19-07-2022
4769    19-07-2022
4770    19-07-2022
4771    19-07-2022
4772    19-07-2022
Name: DataFinalizacao, Length: 4773, dtype: object
[2022-08-18 18:55:11,481] {python.py:173} INFO - Done. Returned value was: None
[2022-08-18 18:55:11,536] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=elastic_desk, task_id=popula_elastic_finalizados, execution_date=20220818T185000, start_date=20220818T185503, end_date=20220818T185511
[2022-08-18 18:55:11,658] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-18 18:55:11,750] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
